apps/docs-app/sidebars.js

```diff
@@ -172,6 +172,17 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: 'category',
+      label: 'Experimental',
+      items: [
+        {
+          type: 'doc',
+          id: 'experimental/sfc/index',
+          label: 'Analog SFCs',
```

Comment:
@brandonroberts  feel like the horse has already left the gate with this one, but any chance of going with "Analog Components" instead of "Analog SFCs", my reasoning: it's not SFC specific, SFC isn't the unique part/value prop of the format, and when this format was specifically SFC only it was this aspect that attracted like 70%+ of the negative feedback on the video comments - more people might give it a shot if they don't assume that they can't do multi file components

Reply:
Yea, I took some inspiration from Vue docs on this one. They still refer to them as SFCs even though you can still do separate template/styles. I think we can use either term, but docs-wise, using SFCs seems more clear to me.

https://vuejs.org/api/sfc-spec.html#src-imports